pkg/flagset/frontend.go

```diff
@@ -89,6 +89,23 @@ func FrontendWithConfig(cfg *config.Config) []cli.Flag {
 			Destination: &cfg.Reva.TransferSecret,
 		},
 
+		// OCDav
+
+		&cli.StringFlag{
+			Name:        "ocdav-files-namespace",
+			Value:       "/home/",
+			Usage:       "Namespace prefix for the /dav/files endpoint",
+			EnvVars:     []string{"OCDAV_FILES_NAMESPACE"},
```

Comment:
This will ultimately go into ocis-webdav - right?

shouldn't it be named WEBDAV_* ...

Reply:
in the end, yes. hmmm ... k will use a single `WEBDAV_JAIL`

Reply:
We need to point the two dav endpoints to different places in the global namespace, which is why I created https://github.com/owncloud/ocis-reva/pull/77